tests/integration/src/test/java/org/apache/pulsar/tests/integration/io/sources/AvroKafkaSourceTest.java

```diff
@@ -19,11 +19,11 @@
 package org.apache.pulsar.tests.integration.io.sources;
 
 import com.google.gson.Gson;
+import dev.failsafe.Failsafe;
```

Comment:
Can we move to Awaiatility?

Also, are we depending on the classes of a maven plugin ?

Reply:
good idea. I pushed the changes